worldedit-bukkit/build.gradle.kts

```
@@ -58,6 +53,9 @@ dependencies {
     "compileOnly"(libs.paperApi) {
         exclude("junit", "junit")
     }
+    "testImplementation"(libs.paperApi) {
+        exclude("junit", "junit")
+    }
```

Can we move these up and change to:
```suggestion
    "localImplementation"(libs.paperApi) {
        exclude("junit", "junit")
    }
```